lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -28,27 +57,148 @@
                                this.$el.text(this.text);
                            }
                        });
-                       this.tabbedView = new TabbedView({
-                           tabs: [{
-                               title: gettext('My Teams'),
-                               url: 'teams',
-                               view: new TempTabView({text: 'This is the new Teams tab.'})
-                           }, {
-                               title: gettext('Browse'),
-                               url: 'browse',
-                               view: new TopicsView({
-                                   collection: options.topicCollection
+                       this.topicsCollection = new TopicCollection(
+                           this.topics,
+                           {url: options.topics_url, course_id: this.course_id, parse: true}
+                       ).bootstrap();
+                       this.mainView = this.tabbedView = new ViewWithHeader({
+                           header: new HeaderView({
+                               model: new HeaderModel({
+                                   description: gettext("Course teams are organized into topics created by course instructors. Try to join others in an existing team before you decide to create a new team!"),
```

Comment:
@catong thanks. I'd like to make the wording around "Consider joining an existing team before creating a new one." Learners should only be creating teams if they don't find one that meets their learning goals or needs. I don't know how we incorporate that without sounding super-mean about it, but I think it would be valuable.